src/main/webapp/i18n/en/student-dashboard.json

```diff
@@ -156,8 +156,8 @@
                 "practice": "Practice",
                 "faq": {
                     "header": "Postings",
-                    "show": "Show replies",
-                    "hide": "Hide replies",
+                    "show": "Show postings",
+                    "hide": "Hide postings",
```

Comment:
On a second inspection, it looks like we didn't rename all occurrences (notice the "Show replies"):
![grafik](https://user-images.githubusercontent.com/72132281/121421210-35bb8780-c96e-11eb-893b-c61fb7a51663.png)

The german translation has the same problem.

Reply:
I'll fix it!

Reply:
solved via a416280